python/servo/build_commands.py

```diff
@@ -244,6 +246,38 @@ def build(self, target=None, release=False, dev=False, jobs=None,
             env['OPENSSL_LIB_DIR'] = openssl_dir
             env['OPENSSL_INCLUDE_DIR'] = path.join(openssl_dir, "include")
             env['OPENSSL_STATIC'] = 'TRUE'
+            # Android builds also require having the gcc bits on the PATH and various INCLUDE
+            # path munging if you do not want to install a standalone NDK. See:
+            # https://dxr.mozilla.org/mozilla-central/source/build/autoconf/android.m4?q=android.m4&redirect_type=direct#139-161
+            os_type = platform.system().lower()
+            cpu_type = platform.machine().lower()
+            if os_type == "linux":
+                if cpu_type in ["i386", "i486", "i686", "i768", "x86"]:
```

Comment:
You only need to have this block once to find the host suffix (e.g. `x86_64`), and then can append it to `os_type + '-'` directly.